src/main/scala/com/campudus/tableaux/router/auth/permission.scala

```diff
@@ -0,0 +1,340 @@
+package com.campudus.tableaux.router.auth
+
+import com.campudus.tableaux.UnauthorizedException
+import com.campudus.tableaux.database.domain.{ColumnType, Table}
+import com.campudus.tableaux.helper.JsonUtils.asSeqOf
+import org.vertx.scala.core.json._
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+
+object RoleModel {
```

Comment:
Here are a lot of classes, case classes, objects, companion objects, sealed traits etc. in one .scala file.
Wouldn't it be better to split this up in separate files?